catalog/app/containers/Bucket/Meta.tsx

```diff
@@ -0,0 +1,104 @@
+import cx from 'classnames'
+import * as R from 'ramda'
+import * as React from 'react'
+import * as M from '@material-ui/core'
+
+import JsonDisplay from 'components/JsonDisplay'
+import AsyncResult from 'utils/AsyncResult'
+
+import Section, { SectionProps } from './Section'
+
+const useHeadCellStyles = M.makeStyles((t) => ({
+  root: {
+    borderRight: `1px solid ${t.palette.divider}`,
+    width: t.spacing(20),
+  },
+}))
+
+interface HeadCellProps {
+  children: React.ReactNode
+  className?: string
+  title: string
+}
+
+const HeadCell = ({ className, children, title }: HeadCellProps) => {
+  const classes = useHeadCellStyles()
+  return (
+    <M.TableCell className={cx(classes.root, className)} component="th" scope="row">
```

Comment:
maybe vertically align this to the top? seems like it would look better that way